File: 217-contains-duplicate/217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        
        tmp = {}
        
        for num in nums:
            
            if tmp.get(num,0) == 1:
                return True
            tmp[num] = tmp.get(num,0) + 1
            
        return False

File: 217-contains-duplicate/217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    
    const hashmap = new Map;
    
    for(let num of nums){
        num = num.toString()
        if(hashmap.has(num)){
            return true
        };
        hashmap.set(num,1)
        
    };
    return false
};